lib/ext/heroku/command/addons.rb

```diff
@@ -439,9 +464,12 @@ def open
         end
       when 1 then
         addon_to_open = matches.first
-        launchy("Opening #{addon_to_open} for #{app}", app_addon_url(addon_to_open))
+        launchy("Opening #{addon_to_open["name"]} for #{app}", addon_to_open["web_url"])
```

Comment:
Yeah I should say that my CLI usage was not the future-proof version, but I meant to highlight the CLI output. Perhaps we display something like

```
Opening swimming-quietly-8857 (heroku-postgresql) for minasan... done
```

or

```
Opening heroku-postgresql (swimming-quietly-8857) for minasan... done
```

For clarity